introduction/what-is-a-research-design.qmd

```diff
@@ -7,53 +7,51 @@
 
 ## MIDA: The four elements of a research design \index{MIDA!Definition}
-
 Research designs share in common that they all have an inquiry *I*, a data strategy *D*, and an answer strategy *A*. Less obviously, perhaps, these three elements presuppose a model *M* of how the world works. We refer to the four together as *MIDA*. 
 
 We think of *MIDA* as having two sides. *M* and *I* form the theoretical half, comprising your beliefs about the world and your target of inference. *D* and *A* form the empirical half, comprising your strategies for collecting and summarizing information. The theoretical side sets the research challenges for you to overcome and the empirical side captures your responses to those challenges.^[We call *M* and *I* the theoretical half because specifying them requires conceptualizing contexts, imagining processes, and posing questions. We call *D* and *A* the empirical half because it describes the empirical strategies. We recognize of course that the theories on the *MI* side may be very thin and that strategies on the *DA* side must be theoretically motivated.] 
 
-@fig-midaplot shows how these four elements of a design relate to one another, how they relate to real world quantities, and how they relate to simulated quantities. We will unpack this figure in the remainder of this chapter, highlighting two especially important parallelisms, first between the upper and lower halves representing actual processes and simulated processes and second between the left (*M*, *I*) and right (*D*, *A*) halves representing these theoretical and empirical sides of research designs.
+@fig-midaplot shows how these four elements of a design relate to one another, how they relate to real-world quantities, and how they relate to simulated quantities. We will unpack this figure in the remainder of this chapter, highlighting two especially important parallelisms, first between the upper and lower halves representing actual processes and simulated processes, and second between the left (*M*, *I*) and right (*D*, *A*) halves representing these theoretical and empirical sides of research designs.
+
 
 \index{MIDA!Illustration}
 
-![The *MIDA* framework. An arrow between two points means that the point at the end of an arrow depends in some way on the one at the start of the arrow. For instance 'the answer you'll get' depends on the dataset you'll get and the answer strategy you specify.](/figures/figure-2-1){#fig-midaplot}
+[The *MIDA* framework. An arrow between two points means that the point at the end of an arrow depends in some way on the one at the start of the arrow. For instance 'the answer you'll get' depends on the dataset you'll get and the answer strategy you specify.](/figures/figure-2-1){#fig-midaplot}
```

Comment:
Removing the ! changed this from a figure to a link. See here for an explanation of figures: https://quarto.org/docs/authoring/figures.html - pls review.

Reply:
Sorry! thought the ! was my typo so I deleted it, now it's there again